akka-stream/src/main/scala/akka/stream/scaladsl/FlowWithContext.scala

```diff
@@ -65,6 +65,14 @@ final class FlowWithContext[-In, -CtxIn, +Out, +CtxOut, +Mat](delegate: Flow[(In
   override def withAttributes(attr: Attributes): FlowWithContext[In, CtxIn, Out, CtxOut, Mat] =
     new FlowWithContext(delegate.withAttributes(attr))
 
+  /**
+   * Context-preserving variant of [[akka.stream.scaladsl.Flow.mapMaterializedValue]].
+   *
+   * @see [[akka.stream.scaladsl.Flow.mapMaterializedValue]]
+   */
+  def mapMaterializedValue[Mat2](f: Mat => Mat2): FlowWithContext[In, CtxIn, Out, CtxOut, Mat2] =
```

Comment:
I wonder if this belongs in `FlowWithContextOps` rather than `FlowWithContext` itself?

Reply:
Originally, I wanted add it into 'FlowWithContextOps' but to perform 'mapMaterializedValue' I need something that already contain materialized value and I find this something only into 'FlowWithContext'. Plus, original 'mapMaterializedValue' for 'Flow' placed into 'Flow' not into 'FlowOps' or 'FlowOpsMat'